### DISCUSSION TITLE
Schema Evolution - (Adding new Column to existing Delta table/ Adding new element to existing struct column in rust/deltalake

### DISCUSSION THREAD

**Discussion by 144853** [delta-io/delta-rs#1762] [Q&A]
Hi Guys., I am trying to do schema evolution for one of the use case using rust/delta on Azure stack. I wan not able to find the api's or examples which will lead me to it. Appreciate any help or guidance. Thanks

**Comment by roeap** [comment]
Hi @144853, right now this is unfortunately not possible with delta-rs.

**Comment by rtyler** [comment]
#2024 has the first pass on adding new columns to the `RecordBatchWriter`.  Depending on your write path, you may not yet benefit from that work
